src/config.rs

```diff
@@ -1472,6 +1475,19 @@ impl Config {
             .map(|path| path.into())
     }
 
+    #[cfg(windows)]
```

Comment:
We shouldn't be conditionally compiling this entire method; XDG should be the default on Windows as well. Perhaps the final fallback path needs to be computed per-platform, though.

Reply:
The [XDG](https://github.com/whitequark/rust-xdg) library does not support windows which makes sense, because it's only applicable for UNIX.

I did find [directories-rs](https://github.com/soc/directories-rs) which has specific windows support.

Reply:
Interesting; maybe we should have a follow-up issue to switch over.

Reply:
I've created an [issue](https://github.com/zacps/alacritty/issues/27) on my fork which will be copied over when we merge.